### DISCUSSION TITLE
Wandercall assets opened to Flare community

### DISCUSSION THREAD

**Discussion by clintbellanger** [flareteam/flare-game#854] [General]
Flare team and community,

Today I made my "wandercall assets" project Public on Github:
https://github.com/clintbellanger/wandercall-assets

Many of these polished assets made their way to OpenGameArt and Sketchfab, but some are yet unpublished and quite messy.

These are Blender 2.7x compatible files similar to the rest of Flare's original assets. If I were to pick up Wandercall again I would want to convert all of this tooling and workflow to modern Blender, along with using PBR textures etc. (Kudos to any community folks already attempting this conversion).

A project the scope of Wandercall is just too large an undertaking for my schedule these days. Hopefully someone will find some of these assets useful!

**Comment by dorkster** [comment]
Very cool! There's some really high quality stuff in there.

P.S. I moved this to the discussions section. Hope you don't mind.

**Reply by WithinAmnesia** [reply]
That plate mail is pretty cool; don't you think? If only we had some talented animators for those new assets hmm. If we only had a talented [custom armour animator](https://opengameart.org/forumtopic/heresy-a-flare-mod-first-act-released) XD. One can dream.

**Comment by WithinAmnesia** [comment]
Thank you Clint! All hail Flare and Wandercall XD.
